### ISSUE TITLE
Add tree metadata browser to layout.

### ISSUE THREAD

**Issue by danielballan** [Xi-CAM/Xi-cam#18]
If I understand correctly, the metadata tree browser (which I remember seeing when it was first created) is currently distributed in this core package but not included in the layout by default. To use it, someone to design and install a Xi-CAM GUI Plugin which imports that code from Xi-CAM core and places it in their custom GUI widget. (Right?)

Alternatively, Xi-CAM could add it to its default layout. I suggested on the Pilot call today that the Preview pane might be made into a tabbed widgets, where the image preview was one tab and the metadata tree was another. Is this worth pursuing?

**Comment by ronpandolfi** [comment]
Yes, something like that idea will be useful. I'm only hesitant about blocking out an appropriate size. The preview pane is small, squarish.

Lets try out integrating it into more GUIPlugins first, starting with CatalogViewer.

Edit: You'd add it the [GUILayout of CatalogViewer](https://github.com/Xi-CAM/Xi-cam.CatalogViewer/blob/5271d61b081d637d1ceb489796fdc96e276cf908/xicam/catalog_viewer/__init__.py#L19) and then connect up `.appendCatalog` to it.
